src/tasks.coffee

```diff
@@ -117,9 +118,11 @@ processNextTaskRound = (task, callback) ->
 
       promises.push defer.promise
 
-  task.save ->
```

Comment:
This can actually be left in: all it does is immediately save the task so as to mark the transactions that just got started as 'Processing'. But it's not too big a deal, it only affects the display on the console.